diff --git a/index.js b/index.js
index 709991da99b8b5cd9c5f4d81774d874da087a20c..4a49c7f4e1322413e5e4b9a7e0fe6fefd6af5695 100644
--- a/index.js
+++ b/index.js
@@ -1,13 +1,13 @@
 const path = require("path");
 
 function isParentFolder(relativeFilePath, context, rootDir) {
-  const absoluteRootPath = path.join(context.getCwd(), rootDir);
-  const absoluteFilePath = path.join(path.dirname(context.getFilename()), relativeFilePath)
+  const absoluteRootPath = path.join(context.cwd, rootDir);
+  const absoluteFilePath = path.join(path.dirname(context.physicalFilename), relativeFilePath)
 
   return relativeFilePath.startsWith("../") && (
     rootDir === '' ||
     (absoluteFilePath.startsWith(absoluteRootPath) &&
-      context.getFilename().startsWith(absoluteRootPath))
+      context.physicalFilename.startsWith(absoluteRootPath))
   );
 }
 
@@ -29,8 +29,8 @@ function getAbsolutePath(relativePath, context, rootDir, prefix) {
     prefix,
     ...path
       .relative(
-        path.join(context.getCwd(), rootDir),
-        path.join(path.dirname(context.getFilename()), relativePath)
+        path.join(context.cwd, rootDir),
+        path.join(path.dirname(context.physicalFilename), relativePath)
       )
       .split(path.sep)
   ].filter(String).join("/");
